spec/spec_helper.rb

```diff
@@ -1,26 +1,21 @@
 #$LOAD_PATH.unshift(File.dirname(__FILE__))
 #$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 require 'coveralls'
-Coveralls.wear!
+Coveralls.wear_merged! # because we run travis on multiple rubies
+
+require 'simplecov'
+SimpleCov.formatter = Coveralls::SimpleCov::Formatter
+SimpleCov.start do
+  add_filter 'spec'
+end
 
-require 'rubygems'
-require 'rspec'
-require 'rspec/core'
 require 'equivalent-xml'
 require 'moab/stanford'
 require 'spec_config'
 require 'ap'
 
 RSpec.configure do |config|
   config.before(:all) {fixture_setup}
-  # Use Gherkin DSL syntax in specs for better readability
-  # scenario "<title>" do
-  #   given: <inputs>
-  #   action: <the application does>
-  #   outcome: <to generate this result>
-  #   ...
-  # end
-  config.alias_example_to :scenario
```

Comment:
there are no more scenarios because i don't think anyone uses the api docs or keeps them up to date and this syntax was just confusing.   I believe the true documentation is in the working specs and in the code itself.